ppr-ui/src/components/unitNotes/UnitNoteReview.vue

```diff
@@ -219,6 +220,12 @@ export default defineComponent({
       setMhrUnitNoteRegistration({ key: key, value: val })
     }
 
+    const getCancelledUnitNoteType = (): string => {
+      const cancelledUnitNote: CancelUnitNoteIF = getMhrUnitNote.value as CancelUnitNoteIF
+      return isCancelUnitNote.value
+        ? UnitNotesInfo[cancelledUnitNote?.cancelledDocumentType]?.header : ''
```

Comment:
Just to clarify, we need to return an empty string when it is not a cancel note?

Reply:
I have simplified the code. Thanks for your advice